assets/js/base/components/product-price/index.tsx

```diff
@@ -216,14 +216,14 @@ export interface ProductPriceProps {
 	 * If both `maxPrice` and `minPrice` are set, the component will be rendered
 	 * as a `PriceRange` component, otherwise, this value will be ignored.
 	 */
-	maxPrice?: number | string;
+	maxPrice?: number | string | undefined;
```

Comment:
This part had been changed to prevent a TS error in the following section:
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L59-L76